src/Exports.jl

```diff
@@ -398,26 +516,91 @@ export groebner_basis_hilbert_driven
 export groebner_basis_with_transformation_matrix
 export groebner_polyhedron
 export group
+export gset
 export h_star_polynomial
 export h_vector
 export halfspace_matrix_pair
+export hall_subgroup
+export hall_subgroup_reps
+export has_center, set_center, center
```

Comment:
Apologies for requesting yet another change, but, I'd really prefer if this was as it was before, i.e.:
```suggestion
export center, has_center, set_center
```

Because the relevant bit is `center`, and `set_` / `has_` just qualify it.

I do not wish to cause you excessive work. It should be easy to change this with a simple regex search&replace, so I am happy to make the change myself; just let me know (I prefer to not make changes to your PR without coordinating them with you!)

Reply:
Please feel free to change this (you are most likely quicker than I) and also incorporate the above suggestion - they all look good to me.

My rational to put the `has_` at the beginning was to ensure that all triples (`has_`, `set_`, something) appear grouped together in `Exports.jl`. Maybe this is useful if one wants to inspect the list in the future?

E.g. there are pairs (`has_`, `set_`) with the getter missing (e.g. `export has_exponent, set_exponent`), which one might want to add in the future. But then, of course one can also find them if these pairs are "somewhere" in the `Exports.jl` file...

Your call. Please change it to a form that you like.

Reply:
done. actually we do have `exponent`, we just forgot to export it, as it's a method for `Base.exponent` and thus always available in the REPL. But when importing `Oscar` into a module, this could lead to confusing problems. So I've added that now. I think that also shows that it's valuable to group these together, so that we notice such problems.